app/controllers/late_policies_controller.rb

```diff
@@ -10,7 +11,7 @@ def action_allowed?
       current_user.instructor_id == instructor_id
     end
   end
-
+  # display all late policies
```

Comment:
is it necessary to have these comments on basic CRUD functions? i think any basic Rails dev should understand what new and edit does